operator/config/flux/network_policy.yaml

```diff
@@ -17,10 +17,8 @@ spec:
         - namespaceSelector:
             matchLabels:
               kubernetes.io/metadata.name: default
-        - podSelector:
+          podSelector:
```

Comment:
Is this correct? This **only** allows ingress from workspace pods in the `default` namespace. I believe we still want the `-` in `- podSelector:` since we can have workspace pods in namespaces other than `default`, or whatever namespace value is overridden by `kustomize`.

Reply:
Moved to: https://github.com/pulumi/pulumi-kubernetes-operator/pull/839

I believe this network policy intends to allow traffic from workspace pods in the default namespace only.  At least that's what I wrote in its header a while ago:
> A network policy to allow Pulumi workspaces in the `default` namespace to fetch Flux artifacts from the source-controller in the `flux-system` namespace.

With the dash, I believe the rule would allow traffic from non-workspace pods in the default namespace.

I would agree that opening it up to all namespaces would be convenient but it might be unsafe for the Flux source controller, since there's no other authentication mechanism for artifact URLs.